### ISSUE TITLE
Unobtainable quest Ally of the Netherwing

### ISSUE THREAD

**Issue by kosta777** [Looking4Group/L4G_Core#3014]
Chain of quests up until this one in Shadowmoon Valley is working but the last npc never gives you this one. I noticed since the reward on it is a pretty neat tankadin shield.

**Comment by kosta777** [comment]
http://www.wowhead.com/quest=10870/ally-of-the-netherwing

**Comment by AnonXS** [comment]
Currently not available due to Netherwing Ledge not released and this is the last Quest, to obtain access to it. Will be released in 2.1.

**Comment by kosta777** [comment]
By NetherwingLedge not released I assume you mean some quest chains there, since I did go there to complete this quest "To Netherwing Ledge!" before finishing Ally of the Netherwing Questline.
I would also like to note that Ally of the Netherwing was added in 2.0, as the comments on Wowhead suggest - top comment on any of these quests is from a person who wrote how to finish the questline and at the time of writing he was at Patch 2.0.6. Finishing that quest in 2.0 did not unlock any other quest lines, it was just a last quest from the chain which gives you reward for completing the long chain. Seems sort of weird to remove only the last - "go talk to this guy" quest with reward for completing last 8 quests in a chain.
